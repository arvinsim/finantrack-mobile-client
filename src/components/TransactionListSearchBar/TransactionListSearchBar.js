import styles from './styles.js'

import React, { Component } from 'react'
import { View } from 'react-native'
import { ListItem, SearchBar } from 'react-native-elements'

class TrasnsactionListSearchBar extends Component {
    render() {
        return (
          <View>
            <SearchBar placeholder='Search Transactions...' />
          </View>
        )
    }
}

export default TrasnsactionListSearchBar