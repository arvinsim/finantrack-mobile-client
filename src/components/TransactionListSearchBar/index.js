import TransactionListSearchBar from './TransactionListSearchBar'

export default TransactionListSearchBar