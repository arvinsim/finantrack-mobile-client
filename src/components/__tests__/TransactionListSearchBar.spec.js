import 'react-native'
import React from 'react'
import TransactionListSearchBar from '../TransactionListSearchBar'

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'

test('renders correctly', () => {
  const tree = renderer.create(
    <TransactionListSearchBar />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})