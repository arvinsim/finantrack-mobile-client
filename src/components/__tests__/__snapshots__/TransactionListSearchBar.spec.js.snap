// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#393e42",
          "borderBottomColor": "#000",
          "borderBottomWidth": 1,
          "borderTopColor": "#000",
          "borderTopWidth": 1,
        },
        false,
        undefined,
      ]
    }
  >
    <TextInput
      autoCapitalize={undefined}
      autoCorrect={undefined}
      autoFocus={undefined}
      blurOnSubmit={undefined}
      clear={undefined}
      clearButtonMode={undefined}
      clearTextOnFocus={undefined}
      dataDetectorTypes={undefined}
      defaultValue={undefined}
      editable={undefined}
      enablesReturnKeyAutomatically={undefined}
      inlineImageLeft={undefined}
      inlineImagePadding={undefined}
      isFocused={undefined}
      keyboardAppearance={undefined}
      keyboardType={undefined}
      maxLength={undefined}
      multiline={undefined}
      numberOfLines={undefined}
      onBlur={undefined}
      onChange={undefined}
      onChangeText={undefined}
      onContentSizeChange={undefined}
      onEndEditing={undefined}
      onFocus={undefined}
      onKeyPress={undefined}
      onLayout={undefined}
      onSelectionChange={undefined}
      onSubmitEditing={undefined}
      placeholder="Search Transactions..."
      placeholderTextColor="#86939e"
      returnKeyLabel={undefined}
      returnKeyType={undefined}
      secureTextEntry={undefined}
      selectTextOnFocus={undefined}
      selectionColor="#86939e"
      selectionState={undefined}
      style={
        Array [
          Object {
            "backgroundColor": "#303337",
            "borderRadius": 3,
            "color": "#86939e",
            "fontSize": 17.5,
            "height": 30,
            "margin": 8,
            "overflow": "hidden",
            "paddingLeft": 26,
            "paddingRight": 19,
          },
          false,
          false,
          false,
          undefined,
        ]
      }
      underlineColorAndroid="transparent"
      value={undefined}
    />
    <Text
      accessible={true}
      allowFontScaling={false}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "#86939e",
            "fontSize": 16,
          },
          Array [
            Object {
              "backgroundColor": "transparent",
              "left": 16,
              "position": "absolute",
              "top": 15.5,
            },
            undefined,
          ],
          Object {
            "fontFamily": "Material Icons",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
        ]
      }
    >
      î¢¶
    </Text>
  </View>
</View>
`;
